src/components/TasksTable/index.jsx

```diff
@@ -61,10 +85,23 @@ const assignments = Object.values(ASSIGNEE);
     justifyContent: 'center',
     paddingTop: theme.spacing.unit,
   },
+  infoButton: {
+    marginRight: theme.spacing.unit,
```

Comment:
Add `marginLeft: -2 * theme.spacing.unit,` as well to align the icon with the column name.